e2e/testdata/fn-render/resource-has-pkgname-prefix/.expected/diff.patch

```diff
@@ -83,32 +84,47 @@ index cbbe597..6b307e8 100644
 -      - image: mysql:5.6 # kpt-set: ${ms-image}:${ms-tag}
 -        name: mysql
 -        env:
-+      - env:
-         - name: MYSQL_ROOT_PASSWORD
-           valueFrom:
-             secretKeyRef:
+-        - name: MYSQL_ROOT_PASSWORD
+-          valueFrom:
+-            secretKeyRef:
 -              name: mysql-pass
-               key: password
-+              name: mysql-pass
-+        image: mysql:5.6 # kpt-set: ${ms-image}:${ms-tag}
-+        name: mysql
-         ports:
-         - containerPort: 3306
+-              key: password
+-        ports:
+-        - containerPort: 3306
++        - env:
++            - name: MYSQL_ROOT_PASSWORD
++              valueFrom:
++                secretKeyRef:
++                  key: password
++                  name: mysql-pass
++          image: mysql:5.6 # kpt-set: ${ms-image}:${ms-tag}
            name: mysql
-         volumeMounts:
+-        volumeMounts:
 -        - name: mysql-persistent-storage
 -          mountPath: /var/lib/mysql
-+        - mountPath: /var/lib/mysql
-+          name: mysql-persistent-storage
++          ports:
++            - containerPort: 3306
++              name: mysql
++          volumeMounts:
++            - mountPath: /var/lib/mysql
++              name: mysql-persistent-storage
        volumes:
-       - name: mysql-persistent-storage
-         persistentVolumeClaim:
+-      - name: mysql-persistent-storage
+-        persistentVolumeClaim:
+-          claimName: mysql-pv-claim
++        - name: mysql-persistent-storage
++          persistentVolumeClaim:
++            claimName: mysql-pv-claim
 diff --git a/wordpress-deployment.yaml b/wordpress-deployment.yaml
-index ee54311..60f1d52 100644
+index d762212..6837328 100644
 --- a/wordpress-deployment.yaml
 +++ b/wordpress-deployment.yaml
-@@ -10,12 +10,13 @@
- apiVersion: v1 # Copyright 2019 Google LLC
+@@ -8,18 +8,19 @@
+ # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ # See the License for the specific language governing permissions and
+ # limitations under the License.
```

Comment:
roundtripping fixes in action here. Note `Copyright 2019 Google....` is no longer appearing here right next to the apiVersion field.